nrf_wifi/fw_if/umac_if/src/tx.c

```diff
@@ -1159,6 +1450,44 @@ static void tx_done_tasklet_fn(unsigned long data)
 }
 #endif /* CONFIG_NRF700X_TX_DONE_WQ_ENABLED */
 
+#ifdef CONFIG_NRF700X_RAW_DATA_TX
+enum nrf_wifi_status (nrf_wifi_fmac_rawtx_done_event_process)(
+		      struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
+		      struct nrf_wifi_event_raw_tx_done *config)
+{
+	enum nrf_wifi_status status = NRF_WIFI_STATUS_FAIL;
+	struct nrf_wifi_fmac_dev_ctx_def *def_dev_ctx = wifi_dev_priv(fmac_dev_ctx);
+
+	if (!fmac_dev_ctx) {
+		goto out;
+	}
+
+	if (!config) {
```

Comment:
Combine the above check too here i.e. `if (!fmac_dev_ctx || !config)`

Reply:
okay

Reply:
done